openshift_tools/monitoring/zagg_sender.py

```diff
@@ -135,7 +131,7 @@ def add_heartbeat(self, heartbeat, host=None):
                                                  )
         self.unique_metrics.append(hb_metric)
 
-    def add_zabbix_keys(self, zabbix_keys, host=None, synthetic=False):
+    def _add_zabbix_keys(self, zabbix_keys, host=None, synthetic=False):
```

Comment:
What I meant is that I kept this so I will remember the names of the functions I need to change in the calling scripts. This is still work in progress.